{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/base.html

```diff
@@ -80,7 +80,7 @@
 
       {% if messages %}
           {% for message in messages %}
-              <div class="alert {% if message.tags %}alert-{{ message.tags }}{% endif %}">{{ message }}<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>
+              <div align="center" class="alert {% if message.tags %}alert-{{ message.tags }}{% endif %}">{{ message }}<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>
```

Comment:
```suggestion
              <div class="text-center alert {% if message.tags %}alert-{{ message.tags }}{% endif %}">{{ message }}<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>
```

Reply:
Thank you for your timely response. I find myself agreeing with you. Regarding the close button, I didn't mean changing its side, I meant that it's not aligned in the vertical plane, as in, it's closer to the bottom of the message.

Reply:
Ah I see. Sorry for not understanding. I'm no HTML expert, vehemently despising every moment of programming it, but I wanted to give some input lol

Reply:
No need for apologies. I understand the sentiment.